docs/build/tools/netrunner.md

```diff
@@ -0,0 +1,189 @@
+# Avalanche Network Runner
+
+The Avalanche Network Runner allows a user to define, create and interact with a network of Avalanche nodes. It can be used for development and testing.
+
+**Note that this tool is not for running production nodes.**
+
+## Installation
+
+The Avalanche Network Runner repository is hosted at [https://github.com/ava-labs/avalanche-network-runner](https://github.com/ava-labs/avalanche-network-runner).
+
+That repository's README details the tool.
+
+Clone the repository with:
+
+```
+git clone https://github.com/ava-labs/avalanche-network-runner.git
+```
+
+Unless otherwise specified, file paths given below are relative to the root of this repository. 
+
+## Usage
+
+The basic pattern for using the Avalanche Network Runner is:
+
+* Define the network
+* Start the network
+* Interact with it
+* Shutdown
+
+The [api](https://github.com/ava-labs/avalanche-network-runner/tree/main/api) package contains client code that allows the user to make API calls to Avalanche nodes within the network.
+
+Please find examples of usage in the [examples](https://github.com/ava-labs/avalanche-network-runner/tree/main/examples) subdirectory.
+
+## Supported Backends
+
+As you can see in the README, a `Network` interface is exposed. There are two implementations of this interface that you may use. One is the [local](https://github.com/ava-labs/avalanche-network-runner/tree/main/local) implementation where each node runs in an isolated process on your machine. The other is the [Kubernetes](https://github.com/ava-labs/avalanche-network-runner/tree/main/k8s) implementation where each node runs in its own Kubernetes pod.
+
+## Local Backend
+
+The simplest and most straightforward way to use the Avalanche Network Runner is to use the local implementation. When you create a network with this tool, each node runs in a process on your machine. **Please note that the maximum size and performance of this network depend on your machine's specifications.**
+
+An example can be found at [examples/local/main.go](https://github.com/ava-labs/avalanche-network-runner/blob/main/examples/local/main.go). It is a simple setup which creates a network of 5 nodes, and performs some simple API calls.
```

Comment:
needs to update to new location of example, probably the extended example to give more usage context

Reply:
I thought that the previous example remained unchanged, and was the actual example, while the new example was only for an interactive network, and thus imo doesn't need to be added here.

Reply:
The previous exampled moved to https://github.com/ava-labs/avalanche-network-runner/blob/main/examples/local/indepth/main.go